don't actually need linux/compiler.h and sanitized headers dont include it

--- qemu-0.8.2/usb-linux.c
+++ qemu-0.8.2/usb-linux.c
@@ -26,7 +26,6 @@
 #if defined(__linux__)
 #include <dirent.h>
 #include <sys/ioctl.h>
-#include <linux/compiler.h>
 #include <linux/usbdevice_fs.h>
 #include <linux/version.h>
 
